.github/workflows/publish.yml

```diff
@@ -313,13 +311,89 @@ jobs:
               owner: context.repo.owner,
               repo: context.repo.repo,
               release_id: release.data.id,
-              name: "medplum-agent-" + packageJson.version + "-linux",
-              data: await fs.readFileSync("packages/agent/medplum-agent-" + packageJson.version + "-linux")
+              name: "medplum-agent-" + process.env.MEDPLUM_VERSION + "-linux-x64",
+              data: await fs.readFileSync(`packages/agent/medplum-agent-${process.env.MEDPLUM_VERSION}-${process.env.MEDPLUM_GIT_SHORTHASH}-linux-x64`)
             });
             await github.rest.repos.uploadReleaseAsset({
               owner: context.repo.owner,
               repo: context.repo.repo,
               release_id: release.data.id,
-              name: "medplum-agent-" + packageJson.version + "-linux.sha256",
-              data: await fs.readFileSync("packages/agent/medplum-agent-" + packageJson.version + "-linux.sha256")
+              name: "medplum-agent-" + process.env.MEDPLUM_VERSION + "-linux-x64.sha256",
+              data: await fs.readFileSync(`packages/agent/medplum-agent-${process.env.MEDPLUM_VERSION}-${process.env.MEDPLUM_GIT_SHORTHASH}-linux-x64.sha256`)
+            });
+
+  build_agent_linux-arm64:
```

Comment:
Would it be possible to use a matrix strategy rather than duplicating the block?